docs/how_to_guides/msm_estimation_exercise.ipynb

```diff
@@ -0,0 +1,1295 @@
+{
```

Comment:
<p>Please add the constraint on the inadmissibility penalty to <code>respy.interface</code> . This will be removed in the flexible choice set PR and by adding it to the interface, we do not have to alter the notebook later,</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
I added the constraint to all the models now. Is this correct? I can adjust it otherwise.